templates/DataDiscovery/index.html.twig

```diff
@@ -19,11 +19,10 @@
     <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/openlayers/2.13.1/OpenLayers.js"></script>
     <script type="text/javascript" src="//maps.google.com/maps/api/js?v=3&key={{ google_maps_api_key }}"></script>
 
-    <script type="text/javascript" src="{{ asset('build/js/geoviz.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('build/js/data-discovery.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('build/js/tree.js') }}"></script>
```

Comment:
Discovery needs the fos_js_routing lines below, otherwise it generates the wrong path.

Reply:
Discovery works for me, Is it saying that Routing element is not defined or is it generating an incorrect path?

Reply:
But the data-discovery search is breaking because of passing null type argument where the LogAction Event dispatcher expects a string. Fixing it now